src/relay/ir/dataflow_matcher.cc

```diff
@@ -625,7 +674,13 @@ class PatternPartitioner : protected MixedModeMutator {
     for (size_t i = 0; i < group.args.size(); ++i) {
       args.push_back(memo_[group.args[i]]);
     }
-    return Call(group.function, args);
+    Function func = WithAttr(group.function, "Partitioned", String(group.name));
+    if (!attrs_.empty()) {
```

Comment:
I think you can remove this check

Reply:
If I remove it, the begin() method of the underlying object segfaults, sometimes the underlying object can be null. This check ensures that segfault doesn't happen.